Dutch airspace 2019

roeles opened the pull request:
I'll leave my repository online for the next year, so the raw-URL will keep working.
The original file can be obtained through www.gliderpilotshop.nl, but it links towards a dropbox location. I have permission to copy the file for the purpose of sharing it with XCSoar users.

lordfolken commented:
I have created a repository for the download.xcsoar.org repo at xcsoar/mapgen-data-content. In there is https://github.com/XCSoar/xcsoar-data-content/tree/master/airspaces where you can add your file and reference to it from here http://download.xcsoar.org/airspaces/

The problem with github hosting is, is that the files are https and that doesn't work on the kobo (yet).

roeles commented:
@lordfolken Thanks! I made a pull request for that repo, removed the file in this pull request and changed the location in the JSON. Please let me know if you spot any mistakes.

roeles commented:
Build fails because my other pull request was not yet merged... should try again later.
